package.json

```diff
@@ -45,23 +45,22 @@
     "requireindex": "^1.2.0"
   },
   "devDependencies": {
-    "@typescript-eslint/parser": "^4.33.0",
-    "eslint": "^7.31.0",
-    "eslint-remote-tester": "^2.0.1",
+    "@typescript-eslint/parser": "^5.9.1",
+    "eslint": "^8.6.0",
+    "eslint-remote-tester": "^2.1.1",
     "eslint-remote-tester-repositories": "^0.0.3",
-    "kcd-scripts": "^11.2.2",
+    "kcd-scripts": "^12.0.0",
     "typescript": "^4.5.3"
   },
   "peerDependencies": {
-    "eslint": ">=6.8"
+    "eslint": "^6.8.0 || ^7.0.0 || ^8.0.0"
```

Comment:
This makes it more explicit that we're 100% supporting a certain version.

If ESLint v9 gets released, it will only be a small change here together with adding the tests

Reply:
Meanwhile developers would be stuck using ESLint v8 or lower until we were able to make that change, and we've already seen how hard it can be to add tests for a new ESLint version...